dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -58,12 +58,20 @@
 
 {% materialization incremental, adapter='snowflake', supported_languages=['sql', 'python'] -%}
 
-  {% set original_query_tag = set_query_tag() %}
 
   {#-- Set vars --#}
   {%- set full_refresh_mode = (should_full_refresh()) -%}
+  {%- set identifier = model['alias'] -%}
```

Comment:
Is `alias` always populated? Or is it an override and we need to pull the name when the user does not provide an alias?

Reply:
Great callout! I saw this is how we did it in the table materialization, but I also am prompted to reflect deeper and admit I don't know the subtleties of this model value. Now for sake of time and reducing risk, I am instead opting for `this.name` since that's far safer an option. Then again, the model['alias'] pattern is precisely what we're using in the table materialization, so I assume it's working fine anyways.